geozero-cli/src/main.rs

```diff
@@ -88,6 +91,26 @@ async fn transform<P: FeatureProcessor>(args: Cli, processor: &mut P) -> Result<
             Some("jsonl") | Some("geojsonl") => {
                 GeozeroDatasource::process(&mut GeoJsonLineReader::new(filein), processor)
             }
+            Some("parquet") | Some("geoparquet") => {
```

Comment:
The community discourages against `.geoparquet` as a file extension, but I figure it's fine to accept files with the suffix.